examples/run_Hahn_echo.py

```diff
@@ -92,7 +92,8 @@
 )
 data.setaxis("ph1", ph1_cyc / 4)
 data.reorder(["ph1", "nScans", "t2"])
-data.set_prop("postproc_type", "spincore_SE_v1")
+data.set_units("t2", "s")
+data.set_prop("postproc_type", "spincore_SE_v3")
```

Comment:
increment post processing version so that we aren't multiplying by nScans anymore